From: Tomasz Buchert <tomasz@buchert.pl>
Date: Wed, 12 Aug 2015 01:02:56 +0200
Subject: Map /dev/stdout and /dev/stderr to their descriptors.

---
 src/util.cc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/util.cc b/src/util.cc
index 1ef77bf..634a942 100644
--- a/src/util.cc
+++ b/src/util.cc
@@ -658,6 +658,12 @@ std::string numeric_name(const struct sockaddr *sa, socklen_t salen) {
 }
 
 int reopen_log_file(const char *path) {
+  if (strcmp("/dev/stderr", path) == 0) {
+    return dup(STDERR_FILENO);
+  }
+  if (strcmp("/dev/stdout", path) == 0) {
+    return dup(STDOUT_FILENO);
+  }
 #if defined(__ANDROID__) || defined(ANDROID)
   int fd;
 
