--- a/src/shrpx.cc.orig	2014-09-10 17:32:07.000000000 +0200
+++ b/src/shrpx.cc	2014-09-24 16:20:37.000000000 +0200
@@ -1225,9 +1225,7 @@
     mod_config()->argv[i] = strdup(argv[i]);
   }
 
-  char cwd[PATH_MAX];
-
-  mod_config()->cwd = getcwd(cwd, sizeof(cwd));
+  mod_config()->cwd = getcwd(NULL, 0);
   if(mod_config()->cwd == nullptr) {
     auto error = errno;
     LOG(FATAL) << "failed to get current working directory: errno=" << error;
